ext/ExponentialUtilitiesStaticArraysExt.jl

```diff
@@ -0,0 +1,143 @@
+#=
+Based on
+Al-Mohy, A. H., & Higham, N. J. (2011). "Computing the action of the matrix exponential, with 
+an application to exponential integrators". SIAM journal on scientific computing, 33(2), 488-511.
+=#
+module ExponentialUtilitiesStaticArraysExt
```

Comment:
```suggestion
module ExponentialUtilitiesStaticArraysExt
```

The docstring already says it